modules/View.js

```diff
@@ -137,16 +138,87 @@
      * @method prepare
      */
     prepare: function() {
-      if (this.model) {
-        return {
-          model: this.model.toJSON(),
-          view: this.viewState.toJSON()
-        };
-      } else {
-        return {
-          view: this.viewState.toJSON()
-        };
+      return this.__getPrepareFieldsContext();
+    },
+
+    /**
+     * Parses the combined arrays from the defaultPrepareFields array and the prepareFields array (or function
+     * returning an array).
+     *
+     * The default prepared fields are: [ { name: 'view', value: 'viewState' }, 'model' ]
+     *
+     * Prepared fields can be defined in a couple of ways:
+     *   preparedFields = [
+     *     'model',
+     *     { name: 'app', value: someGlobalCell },
+     *     'a value that does not exist on the view',
+     *     { name: 'view', value: 'viewState' },
+     *     { name: 'patientId', value: '_patientId' },
+     *     'readOnly'
```

Comment:
i'd name this `nonCellObject` or `pureObject`

Reply:
done